README.rst

```diff
@@ -125,6 +125,21 @@ ignore-bad-ideas
     it is generally a bad idea.
 
 
+Version control integration
+---------------------------
+
+With `pre-commit <https://pre-commit.com>`_, check-manifest can be part of your
+git-workflow. Add the following to your ``.pre-commit-config.yaml``.
+
+.. code-block:: yaml
+
+    repos:
+    -   repo: https://github.com/mgedmin/check-manifest
+        rev: "0.38"
```

Comment:
Presumably this should be 0.39, because there's no .pre-commit-hooks.yaml in https://github.com/mgedmin/check-manifest/tree/0.38/ ?

Reply:
Fixed.